core_data_modules/data_models/validators.py

```diff
@@ -45,3 +47,17 @@ def validate_utc_iso_string(s, variable_name=""):
     validate_iso_string(s, variable_name)
     assert s.endswith("Z") or s.endswith("+00:00"), "{} not a UTC ISO string".format(variable_name)
     return s
+
+
+def validate_url(s, variable_name="", scheme=None):
```

Comment:
A couple of quick tests would be appreciated, more for documentation than correctness.